sbncode/BeamSpillInfoRetriever/getFOM.cpp

```diff
@@ -0,0 +1,410 @@
+/**
+ * @file sbncode/BeamSpillInfoRetriever/getFOM.cpp
+ * @brief Figure of Merit for BNB Spills using SBND information adapted from MicroBooNE FOM
+ * @author Max Dubnowski (maxdub@sas.upenn.edu or `@Max Dubnowski` on SBN Slack)
+ */
+#include <math.h>
+#include "TH1D.h"
+#include "TFitResultPtr.h"
```

Comment:
It turns out what is really needed is:
```suggestion
#include "TFitResult.h"
```